.golangci.yml

```diff
@@ -80,15 +76,14 @@ linters:
     - godot
     - gofmt
     - goimports
-    - golint
+    - revive
     - gomnd
     - goprintffuncname
     - gosec
     - gosimple # enabled by default
     - govet # enabled by default
     - ineffassign # enabled by default
     - lll
-    - maligned
```

Comment:
I believe having `govet` enabled within the `.golangci.yml` automatically pulls in the suggested fieldalignment check.

Reply:
yea, malign should have reported that as well. appears that `malign` hasn't been working maybe :( i'm ok leaving it out of this PR. could be an argument to leave all the linter changes out and make linter tweaks a separate PR, or did it cause an issue in the PR?

Reply:
> or did it cause an issue in the PR

My repeated annoyance of seeing the failing test was the only reason I pulled in the changes. How about I follow up with another PR for the `fieldalignment` change?

Reply:
sure.. i should say, in general, we do have an unspoken rule to only address linter reported issues if they were introduced by the changes in your PR. otherwise, we have had PRs to just address linter reported issues. ideally, at some point, you don't see them anymore.

for the purposes of this PR, we can just move ahead as-is, in my opinion. i was more curious if that proposed `malign` alternative was explored. we will go through and adjust the linter configuration across the projects soon anyway.